pkg/scheduler/apis/config/v1alpha1/defaults.go

```diff
@@ -103,4 +103,12 @@ func SetDefaults_KubeSchedulerConfiguration(obj *kubeschedulerconfigv1alpha1.Kub
 		defaultBindTimeoutSeconds := int64(600)
 		obj.BindTimeoutSeconds = &defaultBindTimeoutSeconds
 	}
+
+	if obj.PodInitialBackoffDurationSeconds == 0 {
+		obj.PodInitialBackoffDurationSeconds = api.DefaultPodInitialBackoffDurationSeconds
```

Comment:
@ahg-g Put them in the same place as 

https://github.com/kubernetes/kubernetes/blob/dcd0975faaeefd41da1c03e7d7cf53273d011d93/pkg/scheduler/apis/config/v1alpha1/defaults.go#L43-L49